### PR TITLE
dig

### PR DESCRIPTION
**Pull request by sunnya97**

### CODE REVIEW

**Comment by dylanschultzie** [comment]
@sunnya97 ahhh you beat me to it. I was about to submit this to get rid of this: https://github.com/osmosis-labs/osmosis-frontend/issues/187

File: src/stores/root.ts [hunk 1]
```diff
@@ -250,6 +250,13 @@ export class RootStore {
 					spotPriceDestDenom: DenomHelper.ibcDenom([{ portId: 'transfer', channelId: 'channel-0' }], 'uatom'),
 					destCoinId: 'cosmos',
 				},
+				{
+					alternativeCoinId: 'pool:udig',
+					poolId: '621',
```
Comment (dylanschultzie) [inline src/stores/root.ts:255]: Pool 620 is also `dig`: https://app.osmosis.zone/pool/620

Comment (sunnya97) [reply src/stores/root.ts:255]: Seems like an abandoned pool that was created by accident with a 3% swap fee

Comment (dylanschultzie) [reply src/stores/root.ts:255]: For sure. I assumed all pools needed identified.

**Comment by MasterPi-2124** [comment]
Hi, I fixed the wss problem and the website is running seamlessly now.
270816745_963386677931272_2543903101733482876_n

File: src/config.ts [hunk 2]
```diff
@@ -1719,4 +1725,46 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+	{
+		rpc: 'https://rpc-1-dig.notional.ventures',
+		rest: 'https://api-1-dig.notional.ventures',
+		chainId: 'dig-1',
+		chainName: 'Dig',
+		stakeCurrency: {
+			coinDenom: 'DIG',
+			coinMinimalDenom: 'udig',
+			coinDecimals: 6,
+			// coinGeckoId: 'pool:udig',
```
Comment (sunnya97) [inline src/config.ts:1737]:
```suggestion
			coinGeckoId: 'pool:udig',
```

File: src/config.ts [hunk 3]
```diff
@@ -1719,4 +1725,46 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+		bip44: {
+			coinType: 118,
+		},
+		bech32Config: Bech32Address.defaultBech32Config('dig'),
+		currencies: [
+			{
+				coinDenom: 'DIG',
+				coinMinimalDenom: 'udig',
+				coinDecimals: 6,
+				// coinGeckoId: 'pool:udig',
```
Comment (sunnya97) [inline src/config.ts:1749]:
```suggestion
				coinGeckoId: 'pool:udig',
```

File: src/config.ts [hunk 4]
```diff
@@ -1719,4 +1725,46 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+				coinImageUrl: window.location.origin + '/public/assets/tokens/dig.png',
+			},
+		],
+		feeCurrencies: [
+			{
+				coinDenom: 'DIG',
+				coinMinimalDenom: 'udig',
+				coinDecimals: 6,
+				// coinGeckoId: 'pool:udig',
```
Comment (sunnya97) [inline src/config.ts:1758]:
```suggestion
				coinGeckoId: 'pool:udig',
```

**Pull request merged**